Components/grid/grid.css

```diff
@@ -0,0 +1,125 @@
+@import url("../../main.css");
+@import url('../../colors.css');
+@import url('../../typography.css');
+
+
+/* colors for unit borders */
+:root {
+    --color1: #001219;
+    --color2: #005f73;
+    --color3: #3d9397;
+    --color4: #94d28d;
+    --color5: #e9d8a6;
+    --color6: #ee9800;
+    --color7: #ca6702;
+    --color8: #bb3e03;
+    --color9: #ae2012;
+    --color10: #9b2226;
+    --color11: rgba(255, 255, 255, 0.8);
```

Comment:
I will recommend giving meaningful names like ( dark red, light green)  rather than color1, color2 etc. So it can be easily distinguished.

Reply:
👍🏼